### PR TITLE
iframe support

### PR DESCRIPTION
**Pull request by amirhalatzi**

### CODE REVIEW

File: src/sdk/putResources.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ function makePutResources(concurrency = DEFAULT_CONCURRENCY_LIMIT) {
 
     const needMoreResources = runningRender.getNeedMoreResources();
     if (needMoreResources) {
-      for (const resource of rGridDom.getResources()) {
+      const availableReosurces = Array.from(new Set(allResources.concat(rGridDom.getResources())));
```
Comment (amitzur) [inline src/sdk/putResources.js:21]: why is this necessary? `allResources` should contain everything already

Comment (amirhalatzi) [reply src/sdk/putResources.js:21]: For backwards compatibility. I didn't fix all the places that invoke this function
